### PR TITLE
[DOCS] Added a contributing to Docker Extension page

### PR DESCRIPTION
**Pull request by Ambareen09**
**Notes for Reviewers**

- Added a page in contributions for the Meshery Docker extension.
- Added docker hub meshery extension link [reference in comments of #7518] 

This PR fixes a sub-task of #6255 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9485ba9dedd04977af0c895238280da335830075 at: https://643458e1a308c15aa9b303d8--meshery-docs-preview.netlify.app

File: docs/v0.5/search/index.html [hunk 1]
```diff
@@ -1029,6 +1029,18 @@ <h1></h1>
...
...e UI, it will be a good idea to use the included UI development server. You can run the UI development server by running the following command:\n\nmake ui\n\n\nRefer to Contributing to Meshery Server, if needed.\n\n\n  Make sure to have Meshery Server configured, up and running on the default port http://localhost:9081 before proceeding to access and work on the UI server at http://localhost:3000.\n\n\n\n  Any UI changes made now will automatically be recompiled and served in the browser.\n\n\nRunning Cypress integration tests\n\nTo run cypress integration tests, a convenience make target called ui-integration-tests that installs dependencies in /ui and /provider-ui folders as prerequisite and invokes ci-test-integration npm script found in /ui/package.json\n\n   \n     $ make ui-integration-tests\n   \n\nAbove command must be run from Meshery repository's root folder.\n\n\n\nRefer to Meshery Cypress Testing for details of how to contribute and benefit from Meshery Cypress (integration &amp; end-to-end) testing.\n\nRunning Meshery from IDE\n\nAll of the above steps would get the Meshery’s development server running for you to work on in any IDE of your choice.\n\nSuggested Reading"
```
Comment (leecalcote) [inline docs/v0.5/search/index.html:1030]: @Ambareen09, let's forego updates to older versions of the Meshery Docs as they are snapshots in time, frozen at the time of final release of that version.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/v0.5/search/index.html (hunk 1 above)
Comment (Ambareen09) [reply docs/v0.5/search/index.html:1030]: you mean, I should revert this change?

**Comment by l5io** [comment]
🚀 Preview for commit 581b73debfb7a215f6e531362ad837efae80e7d5 at: https://64345d8ddc4a9a67c410042d--meshery-docs-preview.netlify.app

File: docs/v0.5/search/index.html (hunk 1 above)
Comment (Ambareen09) [reply docs/v0.5/search/index.html]: Oh sorry, I didn't realize that. Made the suggested change.

**Comment by l5io** [comment]
🚀 Preview for commit 407ff8e3171585b73016bec0019e4337e5266a9b at: https://6434641a0ffc9802b7e9aeff--meshery-docs-preview.netlify.app

**Comment by goldfishdolphin** [comment]
@leecalcote and @Ambareen09 I would like to know which version of Docker Desktop is used for the documentation. I have installed Docker Desktop 4.18.0, which is slightly different from the one on the Contributing to Docker Extension page. 
#### Docker extension tab at version 4.18
Docker extension tab at version 4.18

#### Contributing to Docker Extension page
Screenshot 2023-04-11 at 1 50 41 pm

**Comment by Ambareen09** [comment]
> @leecalcote and @Ambareen09 I would like to know which version of Docker Desktop is used for the documentation. I have installed Docker Desktop 4.18.0, which is slightly different from the one on the Contributing to Docker Extension page.
> 
> #### Docker extension tab at version 4.18
> Docker extension tab at version 4.18
> 
> #### Contributing to Docker Extension page
> Screenshot 2023-04-11 at 1 50 41 pm

My docker desktop version is 4.17.0

**Comment by leecalcote** [comment]
There's no doubt the that the current screenshots are old. Please do refresh at-will.

**Comment by Ambareen09** [comment]
> There's no doubt the that the current screenshots are old. Please do refresh at-will.

Ok, updating

**Comment by l5io** [comment]
🚀 Preview for commit 6ea3d6081e5da5070ba68f24e8b28c11bdc8a762 at: https://64356a83376df00bdf6616ad--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a2c69b3d1ab36df7ed127dd86fe82e6b771bcb92 at: https://6435785036b4771df821f962--meshery-docs-preview.netlify.app

**Comment by goldfishdolphin** [comment]
Looking good @Ambareen09  👍🏽

**Comment by Ambareen09** [comment]
> @Ambareen09 Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

Sure! :)

**Review by leecalcote — APPROVED** [review]
🚀

**Pull request merged**